src/server/scripts/Northrend/zone_borean_tundra.cpp

```diff
@@ -1169,7 +1169,8 @@ class npc_hidden_cultist : public CreatureScript
                                 uiEventPhase = 2;
                                 break;
                             case NPC_TOM_HEGGER:
-                                Talk(SAY_HIDDEN_CULTIST_3);
+                                if (Player* player = ObjectAccessor::GetPlayer(*me, uiPlayerGUID))
+                                    Talk(SAY_HIDDEN_CULTIST_3, player); 
```

Comment:
as a general rule, let's always use brackets `{` `}` even in 1-line `if` blocks, because this can help preventing mistakes

Reply:
Thanks @FrancescoBorzi . I have added the brackets.